tozny/resource_realm_identity.go

```diff
@@ -155,3 +156,54 @@ func resourceRealmIdentityDelete(ctx context.Context, d *schema.ResourceData, m
 
 	return diags
 }
+func resourceRealmIdentityUpdate(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+	var diags diag.Diagnostics
+	toznySDK, err := MakeToznySDK(d, m)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+	// Check if Password has changed
+	if d.HasChange("password") {
+		// Get the old password
+		oldPassword, _ := d.GetChange("password")
+		realmName := d.Get("realm_name").(string)
+		username := d.Get("username").(string)
+		brokerTargetURL := d.Get("broker_target_url").(string)
+		emailExpiryMinutes := d.Get("recovery_email_ttl").(int)
+		identityID, err := strconv.ParseInt(d.Id(), 10, 64)
```

Comment:
you can use a runtime type assertion like above and save the need to import `strconv` package 
identityID := d.Id().(int), while normally saving imports isn't a weighty concern, code consistency is somewhat worth considering.

Reply:
![Screen Shot 2021-09-13 at 6 01 31 PM](https://user-images.githubusercontent.com/54516136/133176780-60c9ab3b-2d10-4326-9219-b0555f84e508.png)

Reply:
![image](https://user-images.githubusercontent.com/7720874/133176921-5dbc7d2b-3ec0-447e-9422-47fb91d2cd51.png)